### ISSUE TITLE
BibTex not displaying abstract

### ISSUE THREAD

**Issue by afnan** [citation-js/citation-js#98]
I am experimenting with a citation from different databases. If I try to import the following bibtex I do not get abstract in any format I export it to. Why do we have _graph? because I can see abstract was extracted in it. 

BibTex

```
@inbook{doi:10.1002/9781118796443.ch28,
author = {Guevara-Villalobos, Orlando},
publisher = {John Wiley & Sons, Ltd},
isbn = {9781118796443},
title = {Artisanal Local Networks: Game Work and Culture in Independent Game Production},
booktitle = {Handbook of Digital Games},
chapter = {28},
pages = {730-750},
doi = {10.1002/9781118796443.ch28},
url = {https://onlinelibrary.wiley.com/doi/abs/10.1002/9781118796443.ch28},
eprint = {https://onlinelibrary.wiley.com/doi/pdf/10.1002/9781118796443.ch28},
year = {2014},
keywords = {artisanal local networks, community-of-practice (CoP), game cultures, game development, game production, game work},
abstract = {Summary This chapter is particularly concerned with the synergies between independent game production and developers′ game cultures. It relies on a cultural industry approach to game work as cultural work, while engaging with the community-of-practice (CoP) framework to understand the internal dynamics of developers’ artisanal local networks. The chapter attempts to describe and reflect on the daily work practices of independent game developers, with special emphasis on a local network in Cambridge, and to provide an understanding of the microsocial relationships that structure their process of game making, both spatially and procedurally. In addition, it is set to explore the meanings and functions of these relationships, which is driven by the pleasures of work/play as suggested by N. Dyer-Witheford and G. De Peuter and C. O’Donnell as well as shaped by the need to socially ensure access to technical, intellectual, and economic sources of game development.}
}
```

Output
```javascript
0:
DOI: "10.1002/9781118796443.ch28"
ISBN: "9781118796443"
URL: "https://onlinelibrary.wiley.com/doi/abs/10.1002/9781118796443.ch28"
author: [{…}, __ob__: Observer]
chapter-number: "28"
citation-label: "doi:10.1002/9781118796443.ch28"
container-title: "Handbook of Digital Games"
id: "doi:10.1002/9781118796443.ch28"
issued: {date-parts: Array(1)}
page: "730-750"
publisher: "John Wiley & Sons, Ltd"
title: "Artisanal Local Networks: Game Work and Culture in Independent Game Production"
type: "chapter"
_graph: Array(6)
0: {type: "@bibtex/text", data: "↵@inbook{doi:10.1002/9781118796443.ch28,↵author = …length of two decision nodes.},↵year = {2018}↵}↵↵"}
1: {type: "@else/list+object"}
2:
data:
label: "doi:10.1002/jeab.320"
properties:
abstract: "Experimental games have previously been used to study principles of human interaction. Many such games are characterized by iterated or repeated designs that model dynamic relationships, including reciprocal cooperation. To enable the study of infinite game repetitions and to avoid endgame effects of lower cooperation toward the final game round, investigators have introduced random termination rules. This study extends previous research that has focused narrowly on repeated Prisoner's Dilemma games by conducting a controlled experiment of two-player, random termination Centipede games involving probabilistic reinforcement and characterized by the longest decision sequences reported in the empirical literature to date (24 decision nodes). Specifically, we assessed mean exit points and cooperation rates, and compared the effects of four different termination rules: no random game termination, random game termination with constant termination probability, random game termination with increasing termination probability, and random game termination with decreasing termination probability. We found that although mean exit points were lower for games with shorter expected game lengths, the subjects' cooperativeness was significantly reduced only in the most extreme condition with decreasing computer termination probability and an expected game length of two decision nodes."
author: "Krockow, Eva M. and Colman, Andrew M. and Pulford, Briony D."
doi: "10.1002/jeab.320"
eprint: "https://onlinelibrary.wiley.com/doi/pdf/10.1002/jeab.320"
journal: "Journal of the Experimental Analysis of Behavior"
keywords: "Centipede game, random game termination, backward induction, endgame effects, cooperation, reciprocity"
number: "2"
pages: "349-364"
title: "Dynamic probability of reinforcement for cooperation: Random game termination in the centipede game"
url: "https://onlinelibrary.wiley.com/doi/abs/10.1002/jeab.320"
volume: "109"
year: "2018"
__proto__: Object
type: "article"
__proto__: Object
type: "@bibtex/object"
__proto__: Object
3: {type: "@bibtex/object", data: {…}}
4: {type: "@bibtex/object", data: {…}}
5: {type: "@csl/list+object"}
length: 6
```

**Comment by larsgw** [comment]
Thank you for taking the time to report this, and my apologies for not replying earlier. I am working on an update of the BibTeX parser which will add the `abstract` mapping, and this mapping will be included in the next releases whether that release includes the complete update or not.

**Comment by larsgw** [comment]
This should be fixed in `v0.5.0-alpha.10`. If not, please re-open the issue.
